packages/cursorless-vscode-e2e/src/suite/fixtures/recorded/languages/nix/changeArgBat.yml

```diff
@@ -0,0 +1,39 @@
+languageId: nix
+command:
+  version: 6
+  spokenForm: change arg bat
+  action:
+    name: clearAndSetSelection
+    target:
+      type: primitive
+      mark: {type: decoratedSymbol, symbolColor: default, character: b}
+      modifiers:
+        - type: containingScope
+          scopeType: {type: argumentOrParameter}
+  usePrePhraseSnapshot: true
+initialState:
+  documentContents: |-
+    { lib, buildPackages, fetchurl, stdenv }:
```

Comment:
this looks like an import. how come you're calling it "arg"? I'd lean towards just letting "item" handle this one, which should work automatically

Reply:
It is because it is actually function args. A function is designated in nix by the ':'. Most nix files are a just function taking inputs and outputting an attrset.

There is an explicit import expression in nix, bit its part of an assignment.

Reply:
I guess worth noting in this case the real argument to the function is actually the single set {}, but in these cases of a set being passed this way it gets destructured into directly referencable variables within the function, so effectively the sets entries passed in this way become individual args (or at least that's how I'm modeling it here). Open to discussion if this is not a good way to do it.

Reply:
huh that's not now we do it in JS, but JS supports multiple args. For example, in JS:

```ts
function aaa({bbb, ccc}) {}
```

In this case `bbb` and `ccc` are referenceable in the body of `aaa`, but the whole `{bbb, ccc}` is considered an arg. But if destructuring and currying are the only way to have multiple args in nix, and your eye sees these as args, maybe it's fine? What about:

```nix
{aaa, bbb}: ccc:
```

ie currying plus destructuring

Reply:
Ya good point. I took a look at some more complicated nix code than what ya usually working on, and I think probably it will make it too confusing. `item` seems like a better fit. I'll change that so the set contents aren't picked up as args.